internal/restconfig/restconfig.go

```diff
@@ -72,6 +72,8 @@ type Producer struct {
 	deprecatedKubeContextsMessage *string
 	defaultNamespace              *string
 	deprecatedNamespaceFlag       *string
+	prefixedNamespace             *string
```

Comment:
Please make it clear that this is the default namespace. Since multiple prefixes can be defined, this will have to be a map (see `prefixedKubeConfigs`).